src/CommandingOfficers/Commander.java

```diff
@@ -132,6 +134,20 @@ public Commander(CommanderInfo info)
     money = DEFAULTSTARTINGMONEY;
 
     myAbilities = new ArrayList<CommanderAbility>();
+
+    // TODO:
+    // This *really* shouldn't be here, but I ain't sure how best to not do that.
+    // Maybe we should just make some static function to set up this stuff, like the AI/CommanderLibraries?
+    // That has the downfall that there's only one return type from that.
+    // I guess we could have one function that builds the whole unit library, and one that pulls out the buildables and makes the map.
+    // Whatever. This works, for now.
+
+    // Subs gotta dive, man
+    UnitModel sub = getUnitModel(UnitModel.UnitEnum.SUB);
+    UnitModel subsub = new SubSubModel();
+    unitModels.add(subsub);
+    sub.possibleActions.add(new UnitActionType.Transform(subsub, "DIVE"));
+    subsub.possibleActions.add(new UnitActionType.Transform(sub, "RISE"));
```

Comment:
I'm still wondering if it would be better to just put the unit type in the `Transform` action, and query the Commander for the specific model to use. You said you're not sure you like that, but is it worse than this?

Reply:
I suppose just using the type enum would work well enough, and allow creation in the unit's constructor.
I guess I'm just not a huge fan of CO.getUnitType because we have various reasons for a CO to have multiple models of a single type... but I don't know how to properly fix that.

Reply:
I've already nuked the one case where we had that (unless I'm forgetting something). I'm not too keen to bring that back into style, either.

Reply:
As of the tip of `readResolve`, Muriel and Strong both use CO.`getUnitModel()` for regular functions.
Meridian and MapMaster use it in their constructors.

~~It's also used in Test, but nobody cares about that.~~

Reply:
Sure, but using co.`getUnitModel()` isn't the problem. Having more than one valid unit model of a given type is. We don't have that anymore. To be clear, my suggestion is to also use `getUnitModel()` for `Transform`.

Reply:
Hmm. Okay.
The problem, I suppose, is that we have a whole modifier type for accomplishing... well, exactly that: adding a new unit model type.
And we assume that any given unit model will exist already for any CO, so there's no case where a UnitRemodel can work without creating a two-unit-model problem.

Reply:
`UnitRemodelModifier` isn't currently used anywhere. Given that you already have a different way of transforming units, I'm fine with deleting this one.
I'm not really sure why we didn't use a `Map<UnitType, UnitModel>` to start with - that seems more natural than just a list. I don't much like the idea of allowing COs to have multiple unit models for one type.

Reply:
Actually, I'm wrong. The UnitRemodel could take a unit between two existing types... but I think that perhaps we should then just feed the UnitRemodelMod a destination type rather than a destination model (as you're proposing we do with Transform).

Okay, I think we've perhaps gotten this settled. Should I just make all these changes on this branch?

Reply:
Do eet.